### PR TITLE
feat: add get_org_from_ccx_course_url function

### PR DESCRIPTION
**Pull request by Ian2012**
This PR creates a new function to extract the org from the ccx course URL.

See https://github.com/openedx/aspects-dbt/pull/44 for more information

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0033_ccx_function.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+revision = "0033"
+down_revision = "0032"
+branch_labels = None
+depends_on = None
+on_cluster = " ON CLUSTER '{{CLICKHOUSE_CLUSTER_NAME}}' " if "{{CLICKHOUSE_CLUSTER_NAME}}" else ""
+engine = "ReplicatedReplacingMergeTree" if "{{CLICKHOUSE_CLUSTER_NAME}}" else "ReplacingMergeTree"
+
+old_user_profile_table = "{{ASPECTS_XAPI_DATABASE}}.old_{{ASPECTS_RAW_XAPI_TABLE}}"
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0033_ccx_function.py:15]: You shouldn't need this

File: tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0033_ccx_function.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"""
+Update the get_course_from_url function to support CCX
```
Comment (bmtcril) [inline tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0033_ccx_function.py:2]: This is incorrect now

**Review by bmtcril — APPROVED** [review]
👍

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/607&entry.1484655318=https://github.com/Ian2012&entry.752974735=2024-02-16+16:11&entry.1917517419=2024-02-20+15:27&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**